ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.6.yaml

```diff
@@ -141,7 +141,7 @@ tests:
     workflow: openshift-e2e-libvirt
 - as: ocp-image-ecosystem-remote-libvirt-ppc64le
   cluster: build01
-  cron: 0 22 * * 1
+  cron: 0 2 6 1 *
```

Comment:
This seems to be incorrect

Reply:
hmm. Jan 6th at 2:00am UTC every year. Did I get it wrong?

Reply:
ImageEco jobs will start at 2:00am UTC on Jan 06, 07, 08 09 for OCP 4.6, 4.7, 4.8 and 4.9, in that order.

Reply:
Discussed with Deep and we decided to move 4.6, 4.7 and 4.8 to @yearly ( we will stop the jobs in couple of months), and not to change current job time for 4.9.
So, I corrected my changes accordingly.